testsuite/shared/src/main/java/org/jboss/as/test/integration/domain/management/util/DomainTestUtils.java

```diff
@@ -351,11 +383,25 @@ public static String getServerState(final ModelControllerClient client, final Pa
      * @return {@code true} if the state matches, {@code false} otherwise
      * @throws IOException
      */
-    public static boolean checkState(final ModelControllerClient client, final PathAddress serverAddress, final String state) throws IOException {
+    public static boolean checkServerState(final ModelControllerClient client, final PathAddress serverAddress, final String state) throws IOException {
         final String serverState = getServerState(client, serverAddress);
         return state.equals(serverState);
     }
 
+    /**
+     * Check the suspend state of server.
+     *
+     * @param client        the controller client
+     * @param serverAddress the server config address
+     * @param state         the expected state
+     * @return {@code true} if the state matches, {@code false} otherwise
+     * @throws IOException
+     */
+    public static boolean checkSuspendState(final ModelControllerClient client, final PathAddress serverAddress, final String state) throws IOException {
```

Comment:
This method looks unused anywhere. Is it intentionally kept in the MR or is it just a leftover?

Reply:
I used them along the feature development but they were not longer necesary. I've just removed them since are not used.